### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by harmeshverma01**

### CODE REVIEW

File: task/models.py [hunk 1]
```diff
@@ -18,6 +16,9 @@ class Task(models.Model):
     descriptions = models.TextField()
     status = models.CharField(choices=CHOICES, max_length=20, default='Todo')
     
-    
     def __str__(self) -> str:
-        return self.title
\ No newline at end of file
+        return self.title
+    
+class CreateUser(models.Model):
```
Comment (akashSharma135) [inline task/models.py:22]: This doesn't meant to be pushed.

File: task/serializer.py [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
 class TaskSerializer(serializers.ModelSerializer):
     class Meta:
         model = Task
-        fields = '__all__'
\ No newline at end of file
+        fields = '__all__'
+        
+        
+class CreateSerializer(serializers.ModelSerializer):
```
Comment (akashSharma135) [inline task/serializer.py:11]: remove

File: task/views.py [hunk 3]
```diff
@@ -50,18 +52,33 @@ def get(self, request, id=None):
...
         page_size = request.GET.get('page_size', 50)
         paginator = Paginator(task, page_size)
         serializer = self.serializer_class(paginator.page(page_number), many=True)
         return Response(serializer.data)
-    
    
-                
\ No newline at end of file
+                
+class UserstatusView(APIView):
```
Comment (akashSharma135) [inline task/views.py:68]: Doesn't meant to be pushed. Just for tests.

File: task/views.py [hunk 4]
```diff
@@ -17,14 +16,17 @@ class TaskView(APIView):
...
-        task = get_object_or_404(Task, manager=manager)
-        serializer = self.serializer_class(task, data=request.data)
-        if serializer.is_valid():
-            serializer.save()
-            return Response(serializer.data, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors)
-    
+        user = request.data.get('user')
+        if user == str(request.user.id):
+            return Response(({'message': 'you can not assign task because user and manager should be same'}))
```
Comment (akashSharma135) [inline task/views.py:21]: Correct this message.

File: task/models.py (hunk 1 above)
Comment (harmeshverma01) [reply task/models.py:22]: fixed

File: task/serializer.py (hunk 2 above)
Comment (harmeshverma01) [reply task/serializer.py:11]: fixed

File: task/views.py (hunk 3 above)
Comment (harmeshverma01) [reply task/views.py:68]: fixed

File: task/views.py (hunk 4 above)
Comment (harmeshverma01) [reply task/views.py:21]: fixed

**Pull request merged**